wpilibc/src/main/native/include/frc/kinematics/SwerveModuleState.h

```diff
@@ -25,5 +25,20 @@ struct SwerveModuleState {
    * Angle of the module.
    */
   Rotation2d angle;
+
+  /**
+   * Optimizes the angle of the module to make sure it doesn't 
+   * rotate more than 90 degrees from the current state
+   */
+  SwerveModuleState OptimizeModuleAngle(const SwerveModuleState& state) const {
```

Comment:
It is not clear whether the parameter represents the desired state or the current state. Changing the param name or including an explanation in the doc comment would help.